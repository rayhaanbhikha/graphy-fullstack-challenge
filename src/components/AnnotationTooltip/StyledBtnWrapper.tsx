import styled from "styled-components";

export const StyledBtnWrapper = styled.div`
  display: flex;
  justify-content: flex-end;
  margin: 2px;
`