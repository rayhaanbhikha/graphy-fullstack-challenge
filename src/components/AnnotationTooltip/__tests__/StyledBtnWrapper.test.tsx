import React from 'react';
import { render } from '@testing-library/react'
import { StyledBtnWrapper } from '../StyledBtnWrapper';
import 'jest-styled-components';

describe('StyledBtnWrapper', () => {

  it('should have the default styles', () => {
    const { asFragment } = render(<StyledBtnWrapper />);
    expect(asFragment()).toMatchSnapshot();
  })
});