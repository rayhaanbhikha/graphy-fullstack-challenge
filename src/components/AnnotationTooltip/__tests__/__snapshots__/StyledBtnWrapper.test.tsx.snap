// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledBtnWrapper should have the default styles 1`] = `
<DocumentFragment>
  .c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: end;
  -webkit-justify-content: flex-end;
  -ms-flex-pack: end;
  justify-content: flex-end;
  margin: 2px;
}

<div
    class="c0"
  />
</DocumentFragment>
`;
